Polyhedron/demo/Polyhedron/Plugins/PMP/Point_inside_polyhedron_widget.ui

```diff
@@ -87,6 +87,19 @@
       </layout>
      </widget>
     </item>
+    <item>
+     <spacer name="verticalSpacer">
+      <property name="orientation">
+       <enum>Qt::Vertical</enum>
+      </property>
+      <property name="sizeHint" stdset="0">
+       <size>
+        <width>20</width>
+        <height>40</height>
```

Comment:
Because the Note with the floating text is really ugly.
![capture](https://cloud.githubusercontent.com/assets/3263539/16446255/05c6e302-3de7-11e6-8572-a89d2677b0e0.JPG)